clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/controllers/GoogleLoadBalancerController.groovy

```diff
@@ -73,6 +73,10 @@ class GoogleLoadBalancerController {
       view.account == account && view.region == region
     }
 
+    if (!view) {
+      return []
```

Comment:
I did it this way to be consistent with the entry-point that just takes `name`, and with the behavior of `aws/loadBalancers`.